modelforge/tests/test_training.py

```diff
@@ -1,8 +1,10 @@
-from typing import Type
-
+import os
 import pytest
 
+IN_GITHUB_ACTIONS = os.getenv("GITHUB_ACTIONS") == "true"
+
 
+@pytest.mark.skipif(IN_GITHUB_ACTIONS, reason="Skipping this test on GitHub Actions")
```

Comment:
I'm assuming this just takes too long on CI?

Reply:
it ran out of memory on macos, it wouldn't take too long. I will add a more specific 'skip if on MacOS statmente'